instrumentation/nginx/src/agent_config.cpp

```diff
@@ -21,6 +21,7 @@ static bool SetupOtlpExporter(toml_table_t* table, ngx_log_t* log, OtelNgxAgentC
   auto endpoint_from_env = std::getenv(otel_exporter_otlp_endpoint_env);
 
   if (endpoint_from_env) {
+    config->exporter.use_ssl_credentials = std::string(endpoint_from_env).find("https://") != std::string::npos;
```

Comment:
Please correct me if I am wrong. Don't we need to assign ca_cert_path or ca_cert_string in this case?

Reply:
Only if you want to verify the cert. Which would prevent mitm and server impersonations potentially. This enables you to at least connect to an https endpoint configured by env var... At all.
I'm not looking to become a maintainer. But without this you can't hit endpoints only hosted via https without a config file change.

Reply:
Ok, make sense.